src/libcrun/handlers/krun.c

```diff
@@ -196,6 +197,20 @@ libkrun_configure_vm (uint32_t ctx_id, void *handle, bool *configured, yajl_val
   if (UNLIKELY (ret < 0))
     return crun_make_error (err, -ret, "could not set krun vm configuration");
 
+  krun_set_gpu_options = dlsym (handle, "krun_set_gpu_options");
+
+  if (krun_set_gpu_options == NULL)
+    return crun_make_error (err, 0, "could not find symbol in the krun library");
```

Comment:
would be better no just not call krun_set_gpu_options if this is not available?

Reply:
Yes, I agree that would be better

Reply:
if we make this optional (the gpu options are used only when specified in some config file), then it is fine as it is and must cause an error